.github/workflows/tests.yaml

```diff
@@ -62,6 +62,9 @@ jobs:
 
       - name: Update
         run: |
+          # work around for missing signing key in gdal 3.5.3 docker
```

Comment:
Would you be interested in moving this to a separate PR with a separate conditional step so it only happens with GDAL 3.5.3?

Reply:
I can, not sure if it's worth it, this seems like an issue with gdal docker itself and will probably go away next time base image is rebuilt, but sure.